task-features/daniel/features/labels.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Trello labels API
+    As a user I want to work with labels in the API
+    
+      Scenario: Create and modify label from board
+        Given the user wants to create a label
+        |key  |value   |
+        |board|QI9piQFl|

```

Comment:
in this case it would be advisable to send a more generic name or to imply that a default board will be used
```suggestion
        |board|DEFAULT_BOARD|

```